encoding/_common64.ts

```diff
@@ -114,7 +114,7 @@ export function decode(
 }
 
 function getByte(char: number, alphabet: Uint8Array): number {
-  const byte = alphabet[char]!;
+  const byte = alphabet[char] ?? 64;
   if (byte === 64) { // alphabet.Base64.length
     throw new TypeError(`Invalid Character (${String.fromCharCode(char)})`);
```

Comment:
Not directly related to this change, but can we change this error message to something like:
```
Cannot decode base64 string as it includes invalid character: "${String.fromCharCode(char)}"
```

to align this to our style guide? https://github.com/denoland/std/blob/main/.github/CONTRIBUTING.md#error-messages